services/aur/aur.service.js

```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
   }
 }
 
+class AurPopularity extends BaseAurService {
+  static category = 'rating'
+
+  static route = { base: 'aur/popularity', pattern: ':packageName' }
+
+  static openApi = {
+    '/aur/popularity/{packageName}': {
+      get: {
+        summary: 'AUR Popularity',
+        description: 'Arch linux User Repository Popularity',
+        parameters: pathParams({ name: 'packageName', example: 'dropbox' }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'popularity' }
+
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
```

Comment:
`metric()` is really intended for formatting numbers that can get quite large - it turns thousands into k, millions into M, etc.
Currently the most popular package on AUR has a rating of 38.23. I don't think this really makes sense as a formatter. I think I'd probably just display it as it stands (having been though your `.precision(2)` schema.
Lets also adjust the test assertion (`isMetric`) to match